tox.ini

```diff
@@ -69,3 +70,19 @@ deps =
     build
 commands =
     python -m build .
+
+[testenv:withromancal]
+allowlist_externals =
+    git
+    bash
+deps =
+    pytest-cov
+commands_pre =
+    bash -c "pip freeze -q | grep 'roman_datamodels @' > {env_tmp_dir}/requirements.txt"
+    pip install git+https://github.com/spacetelescope/romancal.git
```

Comment:
My only question is if this should be against the `dev` version or the last stable version? `dev` feels right, but I want to make sure you agree.

Reply:
I agree I think `dev` is most useful here since the versions tend to march along together.